### PR TITLE
Added Free Resources for CP and Programming

### PR DESCRIPTION
**Pull request by real-phoenix**
Please Merge this pull request and add Hacktoberfest Tag. :D

Thankyou , Glad working for you !!!

### CODE REVIEW

**Comment by real-phoenix** [comment]
@closebracket  Sir , please merge my PR #60 under the tag of Hacktoberfest asap , as it'll end in few days ! 
Thanks in Advance :D

File: Programming.md [hunk 1]
```diff
@@ -93,3 +93,17 @@
 8. [CP Algorithms](https://cp-algorithms.com/) - A collection of important algorithms written in C++.
 9. [Atcoder](https://atcoder.jp/) - AtCoder is a programming contest site for anyone from beginners to experts.
 10. [Clist.by](https://clist.by/) - Get updates of upcoming competetive programming contests.
+11. [TopCoder](https://www.topcoder.com/) - All in one site for CP , ML/AI , Developement 
+12. [SPOJ](https://www.spoj.com/) - Beginner friendly website for CP 
+13. [HackerEarth] (https://www.hackerearth.com/) - Great free-site for Developement and Coding 
+14. [Codedrills] (https://codedrills.io/) - Best site for ACM-ICPC Mock Rounds (Indian Regionals and Prelims)
+
+## Mathematics and Programming
```
Comment (closebracket) [inline Programming.md:102]: As per the rule regarding new categories, you should add 5 resources for that category.

File: Programming.md [hunk 2]
```diff
@@ -93,3 +93,17 @@
...
+11. [TopCoder](https://www.topcoder.com/) - All in one site for CP , ML/AI , Developement 
+12. [SPOJ](https://www.spoj.com/) - Beginner friendly website for CP 
+13. [HackerEarth] (https://www.hackerearth.com/) - Great free-site for Developement and Coding 
+14. [Codedrills] (https://codedrills.io/) - Best site for ACM-ICPC Mock Rounds (Indian Regionals and Prelims)
+
+## Mathematics and Programming 
+
+1. [ProjectEuler](https://projecteuler.net/about) - Best site for Maths Problems which are easily solved using programming.
+
+## Useful Codeforces Blogs
```
Comment (closebracket) [inline Programming.md:110]: As per the rule regarding new categories, you should add 5 resources for that category.

File: Programming.md [hunk 3]
```diff
@@ -93,3 +93,17 @@
...
+
+## Mathematics and Programming 
+
+1. [ProjectEuler](https://projecteuler.net/about) - Best site for Maths Problems which are easily solved using programming.
+
+## Useful Codeforces Blogs 
+
+1. [C++Tricks](https://codeforces.com/blog/entry/15643) - Many Tricks are explained well. 
+2. [DSA](https://codeforces.com/blog/entry/13529) - Best blog for resources like Data Structures and Algorithms.
+3. [CP-Blogs](https://www.topcoder.com/thrive/search?tags[]=Competitive%20Programming%20Tutorials) - TopCoders CP Blogs
```
Comment (closebracket) [inline Programming.md]: This resource is very similar to the one on line 96

**Review by closebracket — CHANGES_REQUESTED** [review]
Please see feedback. This will not be approved until changes are made. I would also point out that 7 hours is not long enough before re-requesting a review. Most people have other responsibilities.

**Comment by sbrl** [comment]
By adding the `hacktoberfest-accepted` tag, we can ensure it will still be counted even if it's merged after the deadline.

**Comment by real-phoenix** [comment]
> Please see feedback. This will not be approved until changes are made. I would also point out that 7 hours is not long enough before re-requesting a review. Most people have other responsibilities.


I am working on the required changes.
My Bad Sir !

**Comment by real-phoenix** [comment]
@closebracket  Sir , I have made the desired changes , please have a look at them whenever you are free.

**Pull request merged**